run-test.py

```diff
@@ -838,10 +841,10 @@ def deploy_observer(path, keep_database=False):
            "/../observer", flags="-r", ignore_errors=True)
     ssh(OBSERVER, "cd "+INSPATH +
         "/../observer && GO111MODULE=on make all && mkdir -p .artifacts")
-    scp_to(OBSERVER, "/tmp/insolar-jepsen-configs/observer.yaml",
-           INSPATH+"/../observer/.artifacts/observer.yaml")
-    scp_to(OBSERVER, "/tmp/insolar-jepsen-configs/observerapi.yaml",
-           INSPATH+"/../observer/.artifacts/observerapi.yaml")
+
+    for сqw in "".split("observer observerapi stats-collector migrate"):
```

Comment:
This line doesn't do what you expect.

Reply:
explain pls

Reply:
```
$ bpython
>>> "".split("observer observerapi stats-collector migrate")
['']
```

Also please avoid using Perl idioms in Python code. Just use regular lists.

Reply:
thx